velox/docs/functions/spark/datetime.rst

```diff
@@ -34,6 +34,13 @@ These functions support TIMESTAMP and DATE input types.
     deducted from ``start_date``.
     Supported types for ``num_days`` are: TINYINT, SMALLINT, INTEGER.
 
+.. spark:function:: date_format(timestamp, dateFormat) -> string
+
+    Converts `timestamp` to a string in the format specified by `dateFormat`.
```

Comment:
nit: use `` to refer to a variable.

Reply:
Seems no need to wrap string with it.

Reply:
I was trying to say the \`timestamp\` could be changed as \`\`timestamp\`\`. Same for \`dateFormat\`.

Reply:
Fixed.